.gitignore

```diff
@@ -18,3 +18,4 @@ CMakeLists.txt.user
 
 # VS Code system files
 .vscode/
+default.profraw
```

Comment:
Adding to the list of ignores doesn't remove the file.  We also shouldn't have to ignore this file as it shouldn't appear in the source directory.  The fact that is has turned up in the source directory indicates that at some point something has gone wrong with the building of the library.

Reply:
Done.